templates/teamHistory.html

```diff
@@ -25,7 +25,7 @@
     <div class="table-responsive">
       <table class="table table-striped table-bordered table-condensed" id="team-history-players">
         <thead>
-          <tr><th>Name</th><th title="Position">Pos</th><th title="Games Played">GP</th><th title="Minutes Per Game">Min</th><th title="Points Per Game">Pts</th><th title="Rebounds Per Game">Reb</th><th title="Assists Per Game">Ast</th><th title="Player Efficiency Rating">PER</th><th title="Estimated Wins Added">EWA</th></tr>
+          <tr><th>Name</th><th title="Position">Pos</th><th title="Games Played">GP</th><th title="Minutes Per Game">Min</th><th title="Points Per Game">Pts</th><th title="Rebounds Per Game">Reb</th><th title="Assists Per Game">Ast</th><th title="Player Efficiency Rating">PER</th><th title="Estimated Wins Added">EWA</th><th title="Last Year on Team">Last Yr</th></tr>
```

Comment:
Yeah, I wasn't sure on that either.  Tried to give it a decent tool-tip at least.